app/scripts/add-test-accounts-for-integration-tests.sh

```diff
@@ -24,3 +24,6 @@ $here/cli.sh addRole report.reviewer reports-reviewer
 $here/cli.sh addRole report.reviewer user-manager
 $here/cli.sh addRole report.reviewer admin
 $here/cli.sh addRole report.reviewer reports-reader report:html
```

Comment:
I wanted to make the report reviewer user be logged in for the report integration tests in order to see the changelog report versions. However fetching data from the report api appeared to fail unless the user was added to the modelling group. I have no idea why!!